File: update_config.dart
// Random Dart class generated on 2025-06-20

class DeleteRequest {
  String _name;
  int _value;

  DeleteRequest(this._name, this._value);

  String get name => _name;
  int get value => _value;

  set name(String newName) {
    _name = newName;
  }

  set value(int newValue) {
    _value = newValue;
  }

  Map<String, dynamic> toJson() {
    return {
      'name': _name,
      'value': _value,
    };
  }
}

File: update_config.ts
// Random TypeScript class generated on 2025-10-08

class ProcessValue {
    private value: number;
    private readonly createdAt: Date;

    constructor(initialValue: number = 21) {
        this.value = initialValue;
        this.createdAt = new Date();
    }

    public getValue(): number {
        return this.value;
    }

    public updateValue(newValue: number): void {
        this.value = newValue;
    }

    public getInfo(): object {
        return {
            value: this.value,
            createdAt: this.createdAt.toISOString()
        };
    }
}

export { ProcessValue };
